eth-contracts/contracts/ClaimsManager.sol

```diff
@@ -151,6 +167,8 @@ contract ClaimsManager is InitializableV2 {
      * @param _governanceAddress - address for new Governance contract
      */
     function setGovernanceAddress(address _governanceAddress) external {
+        _requireIsInitialized();
```

Comment:
thinking forward to M10 the 'setGovernanceAddress' etc functions will still have to be _requireIsInitialized while other functions will wrap it and check relevant local addresses as well

Reply:
yep you right